google/auth/pluggable.py

```diff
@@ -132,7 +132,7 @@ def __init__(
         self._credential_source_executable_output_file = self._credential_source_executable.get(
             "output_file"
         )
-        self._tokeninfo_username = kwargs.get("tokeninfo_username", "")  # dummy value
+        self._tokeninfo_username = "" # dummy value. This variable is only used via injection, not exposed to ctor
```

Comment:
Nit: I think `None` is a more appropriate placeholder

Reply:
Then we are going to re-write a bunch of tests since we didn't properly mock the behavior.
This will be updated by a full feature of retrieve token info from the proper endpoint.